app/views/notifications/_feed.html.erb

```diff
@@ -1,21 +1,3 @@
 <% notifications.each do |notification| %>
-  <% present notification do |notif_presenter| %>
-    <li class="notification <%= notification.read? ? '' : 'unread' %>">
-      <div class="body">
-        <%= notif_presenter.icon %>
-
-        <div class="time">
-          <%= notif_presenter.created_at_ago %>
-        </div>
-
-        <div class="title">
-          <%= notif_presenter.render_title %>
-        </div>
-
-        <div class="details">
-          <%= notif_presenter.avatar_with_link(30) %>
-        </div>
-      </div>
-    </li>
-  <% end %>
+  <%= render partial: 'item', locals: { notification: notification } %>
```

Comment:
I think this partial can be called `notification` and this could be just `render notification`